clients-src/modules/ROOT/pages/clients.adoc

```diff
@@ -6,125 +6,76 @@
 
 To work with TypeDB databases we can use one of the standalone software tools developed by Vaticle:
 
 . xref:studio.adoc[TypeDB Studio] -- IDE with GUI and TypeQL syntax highlighting.
-. xref:console.adoc[TypeDB Console] -- powerful CLI tool, most useful for automation.
+. xref:console.adoc[TypeDB Console] -- powerful CLI tool.
+
+Both tools are complete software products that can be used to connect to TypeDB and interact with it. They manage
+connections, sessions and transactions, as well as submitting queries and reading back the responses.
 
-Both tools are complete software products that can be used to connect to TypeDB and interact with it. They take care
-of connection, session and transaction management, automate query sending and process the query response.
+//#todo is this "important"? Or more of "note"?
+[IMPORTANT]
+====
+User-friendly standalone applications also use a lower-level Driver to handle communication with the server.
+
+For example, both TypeDB Studio and TypeDB Console are built on top of the Java TypeDB Driver.
+====
 
 [#_typedb_drivers]
 == TypeDB Drivers
 
-A TypeDB Driver, is a library that implements https://github.com/vaticle/typedb-protocol[TypeDB Client RPC,window=_blank]
-protocol and provides TypeDB Driver API.
+A TypeDB Driver is a library that implements https://github.com/vaticle/typedb-protocol[TypeDB Client RPC,window=_blank]
+protocol. TypeDB Drivers are available for some of the most popular programming languages, allowing you to
+build applications that connect to a TypeDB database easily.
 
-A TypeDB Driver can be used to connect to a TypeDB server. If we are building an application that uses a TypeDB database,
-we would need a TypeDB driver at our application layer to handle the database operations.
-TypeDB Drivers are available for some of the most popular programming languages.
-
-The following TypeDB Drivers are officially supported and actively maintained by the Vaticle. They
-usually support latest TypeDB features and receive continuous bug fixes and improvements.
+The following TypeDB Drivers are officially supported and actively maintained by Vaticle. They
+support the latest TypeDB features and receive continuous bug fixes and improvements.
 
 * xref:java/java-overview.adoc[Java]
 * xref:node-js/node-js-overview.adoc[Node.js]
 * xref:python/python-overview.adoc[Python]
 * Rust (https://github.com/vaticle/typedb-client-rust[Under development,window=_blank])
 
-We also have some community projects for xref:other-languages.adoc[other Languages] and provide support for
-creating a xref:new-driver.adoc[new client].
+There are also xref:other-languages.adoc[community-developed clients]
+(including a https://pypi.org/project/typedb-jupyter/[Jupyter connector,window=_blank])
+and guides for creating xref:new-driver.adoc[new drivers].
 
 === Architecture of a TypeDB Driver
 
-All TypeDB Drivers share a common architecture. A TypeDB Drivers needs a `gRPC` client library to communicate with
```

Comment:
Rolled most of the deleted lines into two sentences, and let the image do more of the work.